data/data-pipeline/data_pipeline/ipython/aggregate_burden_exploration.ipynb

```diff
@@ -0,0 +1,1432 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "029b2e96-5a58-4816-81cc-4f60da27e518",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import pathlib\n",
+    "\n",
+    "%load_ext lab_black"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c20b1785-f802-474d-8aec-423ec574bd05",
+   "metadata": {},
+   "source": [
+    "# Creating an aggregate burden index\n",
+    "\n",
+    "Although we will not be using a aggregate burden index for v1.0 of the CEJST, the USDS team wanted to demonstrate how even duplicating CalEnviroScreen's cumulative index (or a loose interpretation of it) would impact the communities highlighted. \n",
+    "\n",
+    "The data team believes that a threshold methodology has significant limitations that an aggregate or cumulative burden index could remediate, and presents the following as an example of such an index. "
```

Comment:
This is really great! Do you mind adding a short verbal description of the methodology works? Even though it can be reverse-engineered from the code below.

Reply:
Similarly, maybe add a high-level verbal overview of the stats below comparing N to the cumulative burden index, e.g. just for .8 or .65?

Reply:
Re your first point: sure, but I want us to be very explicit that we are pulling from CalEnviroScreen

Re your second point: I am not the most comfortable explicitly doing that -- I think it serves mostly to say, "hey, we tried, y'all!" and is not particularly productive. In addition, because there are so many superficial similarities between the tracts identified by a cumulative burden and the score we use, I think this can open a real can of worms and backfire.